spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -90,18 +90,20 @@ class HyperConformance (implicit cmd: ConformanceConfig,
  *
  * In order to use it in hyperdrive the component needs to be configured in 'ingestion.properties' as follows:
  * {{{
- * transformer.hyperconformance.menas.rest.uri=http://localhost:8080
- * transformer.hyperconformance.dataset.name=example
- * transformer.hyperconformance.dataset.version=1
- * transformer.hyperconformance.report.date=2020-01-29
- * transformer.hyperconformance.report.version=1
- * transformer.hyperconformance.event.timestamp.column=EV_TIME
+ * component.transformer.id.1=[hyperconformance]
+ * component.transformer.class.[hyperconformance]=za.co.absa.enceladus.conformance.HyperConformance
+ * transformer.[hyperconformance].menas.rest.uri=http://localhost:8080
```

Comment:
What about the two dots surrounding the []?

Reply:
The dots are syntactically relevant for the configuration (to select subsets of the config), while the brackets aren't